docs/src/api/class-locatorassertions.md

```diff
@@ -280,8 +280,11 @@ locator = page.locator(".subscribe")
 expect(locator).to_be_checked()
 ```
 
-### option: LocatorAssertions.toBeChecked.timeout = %%-assertions-timeout-%%
+### option: LocatorAssertions.toBeChecked.checked
+* langs: js
```

Comment:
We'll  follow up with other langs, right?

Reply:
I think so, they'll see it in the changes bot.